witchcraft/server_metrics.go

```diff
@@ -130,14 +127,33 @@ func (s *Server) initMetrics(ctx context.Context, installCfg config.Install) (rR
 		localMetricLogger.Metric(metricID, metricType, metric1log.Values(valuesToUse), metricTagsParam)
 	}
 
+	emitAllFn := func() {
+		// emit all metrics a final time on termination
+		if !s.disableGoRuntimeMetrics {
+			metrics.CaptureRuntimeMemStatsOnce(metricsRegistry)
```

Comment:
I'm confused about this -- the still existing logic at line 82 calls `metrics.CaptureRuntimeMemStatsWithContext`, which starts a routine that captures the runtime stats at a given frequency.

This `emitAllFn` is also called in a goroutine at line 147, which is also called periodically. Was the intention to remove the call at line 82?

Reply:
Yes -- this now initializes using the CaptureRuntimeMemStatsFunc updated in https://github.com/palantir/pkg/pull/373 and the function is (only) called below in emitAllFn.